CMakeLists.txt

```diff
@@ -73,7 +135,16 @@ endif()
 option (ENABLE_FFT "Enable FFT transform" OFF)
 if (ENABLE_FFT)
     add_definitions (-DENABLE_FFT)
-    find_package (Heffte 2.2.0 REQUIRED)
+    find_package (Heffte 2.2.0 QUIET)
+    if(NOT Heffte_FOUND)
+        message(STATUS "Using FetchContent for Heffte")
+        FetchContent_Declare(
+             Heffte
+             #DOWNLOAD_EXTRACT_TIMESTAMP
+             GIT_REPOSITORY https://github.com/icl-utk-edu/heffte 
```

Comment:
What version of HeFFTe will this fetch? I think we should be able to specify the version.

Reply:
This will fetch master, which is required for `TestCos1` afaik, right @vmontanaro?

Reply:
Correct.

Reply:
So I guess since there's no release with Cos1, we're gonna have to stick with master.

Reply:
Yes for now that's fine.. But just like passing Kokkos version as an option here also passing Heffte version as an option would be good. It can default to master if none is passed.